AbbyM/Lab1/src/hello_world.cpp

```diff
@@ -0,0 +1,8 @@
+#include "../include/hello_world.h"
```

Comment:
please do not use relative includes, see here about how cmake handles your include path setup for you:

https://cliutils.gitlab.io/modern-cmake/chapters/basics.html#targets-are-your-friend